scripts/config.py

```diff
@@ -76,12 +76,10 @@ def realfull_adapter(_name, _value, _active):
     'MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH', # interacts with CTR_DRBG_128_BIT_KEY
     'MBEDTLS_AES_USE_HARDWARE_ONLY', # hardware dependency
     'MBEDTLS_BLOCK_CIPHER_NO_DECRYPT', # incompatible with ECB in PSA, CBC/XTS/NIST_KW
-    'MBEDTLS_CTR_DRBG_USE_128_BIT_KEY', # interacts with ENTROPY_FORCE_SHA256
```

Comment:
This pull request eliminates `MBEDTLS_CTR_DRBG_USE_128_BIT_KEY` from mbedtls code, but not from the build scripts. Shouldn't the warning about using 128-bit CTR_DRBG be removed from the build scripts?

That warning is obsolete anyway because enabling a 128-bit DRBG now has to use a very explicit request: defining the RNG strength to be 128.